osbuild/pipeline.py

```diff
@@ -17,6 +17,11 @@
 BOLD = "\033[1m"
 
 
+def cleanup(*objs):
+    """Call cleanup method for all objects, filters None values out"""
+    _ = map(lambda o: o.cleanup(), filter(None, objs))
```

Comment:
`cleanup()`? POSIX developers stand in awe of this exquisite naming choice! Apart from that, looks all good.